core/server/common/src/main/java/alluxio/master/journal/Journal.java

```diff
@@ -11,19 +11,19 @@
 
 package alluxio.master.journal;
 
+import alluxio.master.journal.options.JournalReaderCreateOptions;
```

Comment:
I think the options should be called based on the method they are passed into ... so `GetReaderOptions` and `GetWriterOptions`